src/7.1_HW/AccountingDepartment.ts

```diff
@@ -0,0 +1,27 @@
+import { Department } from './Department';
+import { Status } from './Employee';
+
+export class AccountingDepartment extends Department {
+  balance: number;
+
+  constructor(budget: Budget) {
+    super('John', 'Accounting', [], budget);
+    this.balance = this.calculateBalance();
+  }
+
+  addToBalance(amount: number) {
+    this.balance += amount;
+  }
+
+  removeFromBalance(amount: number) {
+    this.balance -= amount;
+  }
+
+  paySalaries() {
```

Comment:
Please, do not forget to add type annotation for return function type explicitly (for all three methods).